doc/en/user/source/extensions/css/workshop/raster.rst

```diff
@@ -0,0 +1,354 @@
+.. _extensions_css_cartography_raster:
+
+Rasters
+=======
+
+Finally we will look at using CSS styling for the portrayal of raster data.
+
+.. figure:: img/RasterSymbology.svg
+
+   Raster Symbology
+    
+
+Review of raster symbology:
+
+* Raster data is **Grid Coverage** where values have been recorded in a regular array. In OGC terms a **Coverage** can be used to look up a value or measurement for each location.
+  
+* When queried with a "sample" location:
+  
+  * A grid coverage can determine the appropriate array location and retrieve a value. Different techniques may be used interpolate an appropriate value from several measurements (higher quality) or directly return the "nearest neighbor" (faster).
+  
+  * A vector coverages would use a point-in-polygon check and return an appropriate attribute value.
+  
+  * A scientific model can calculate a value for each sample location
+  
+* Many raster formats organize information into bands of content. Values recorded in these bands and may be mapped into colors for display (a process similar to theming an attribute for vector data).
+  
+  For imagery the raster data is already formed into red, green and blue bands for display.
+  
+* As raster data has no inherent shape, the format is responsible for describing the orientation and location of the grid used to record measurements.
+
+These raster examples use a digital elevation model consisting of a single band of height measurements. The imagery examples use an RGB image that has been hand coloured for use as a base map.
+
+Reference:
+
+* `Raster Symbology <../properties.html#raster-symbology>`_ (User Manual | CSS Property Listing )
+* `Rasters <../cookbook/raster.html>`_ (User Manual | CSS Cookbook );
+
+The exercise makes use of the ``usgs:dem`` and ``ne:ne1`` layers.
```

Comment:
I think the setup file I added (sort of) has this, but I remember UGS being a pain to get...